.eslintrc.json

```diff
@@ -1,17 +1,26 @@
 {
   "root": true,
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "warn"
```

Comment:
I have marked these issues as a warning, for now, to keep this PR small and consumable. It would be great to raise PRs to address each of these issues later.